File: 19 Co wyświetli podany kod/treść.py
from math import prod

y = 7
y -= 6
y += 9
y += ord("c")
y *= ord(chr(7))
y //= 99
y *= prod(range(999))

if y == 6:
    print("Putra")
elif y == 4:
    print("Burton")
elif y == 2:
    print("Edward")
elif y == 0:
    print("Combs")

File: 19 Co wyświetli podany kod/treść.cpp
#include <iostream>

using namespace std;

int main()
{
    int y = 7;
    y -= 6;
    y += 9;
    y += (int)'c';
    y *= (int)(char)7;
    y /= 99;
    for (int i = 0; i < 999; i++)
        y *= i;

    if (y == 6)
        cout << "Putra";
    else if (y == 4)
        cout << "Burton";
    else if (y == 2)
        cout << "Edward";
    else if (y == 0)
        cout << "Combs";
}
